_data/distros.yml

```diff
@@ -48,4 +48,6 @@
 
 - slemicro:
   name: SUSE Linux Enterprise Micro
+  supported: true
+  tested: true
```

Comment:
@garrett : Did you have some specific interpretation of "supported" and "tested" in mind? In practice, these are really the same for us.

"supported" is fungible -- We don't support it upstream, as we don't have experience with SLE Micro. However, we definitively don't test it upstream. There is the [seed of an OpenSUSE image](https://github.com/cockpit-project/bots/pull/3591), but this will require a lot of work still before the tests actually run there. So for now I'd leave this as false (or absent).

Reply:
I was assuming _supported_ and _tested_ comes from whoever includes Cockpit or parts of it in their distro. If so, I can assure you we're clearly testing Cockpit with SLE Micro as we offer enterprise grade support to customers.

Reply:
@rsimai : Right, the page does not really define what these terms mean. So far these have been from upstream's point of view, but I think we need to clarify this. @garrett , WDYT?

Reply:
I think it was: Officially supported by the distro, tested in our testing framework?

We can always change what's on the website to make things more clear, of course.

Reply:
Whatever you consider useful. I find it important to show that we not only include Cockpit but fully support it. Maybe there's a way to add a note "testing done by vendor" or such, if SLE Micro is an exception? Else, is there anything how we could support you in adding the required testing to your framework?

Reply:
don't want to put any pressure but since it's open for a while, can we progress this request? I'm not sure if I'm supposed to make a change, and if so, what exactly? Thanks!

Reply:
Apologies for this taking so long; we had a high priority fix and I've been concentrating on dark mode too.

It's still not clear what it means, but I've been talking with @martinpitt about it in IRC. I think he can clarify. And I think it means we might need to change the page too?